build.gradle

```diff
@@ -264,7 +264,7 @@ dependencies {
     compile 'org.jooq:jool-java-8:0.9.14'
     compile 'com.github.hmcts:ccd-case-document-am-client:1.7.1'
 
-    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.9.1'
+    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.10.18-prerelease'
```

Comment:
This needs to be updated to a full release version of the hmcts/ccd-test-defintions before going into master. see https://github.com/hmcts/ccd-test-definitions/pull/32.